### ISSUE TITLE
[FORMATTING] Qualify not getting its own line

### ISSUE THREAD

**Issue by lulufredo** [sql-formatter-org/sql-formatter#643]
**Input data**

Which SQL and options did you provide as input?

```sql
WHERE a.b IS NOT NULL QUALIFY ROW_NUMBER() OVER ()...
```

**Expected Output**

```sql
WHERE 
      a.b IS NOT NULL 
      QUALIFY ROW_NUMBER() OVER ()...
```

**Actual Output**

```sql
WHERE 
      a.b IS NOT NULL QUALIFY ROW_NUMBER() OVER ()...
```

**Usage**

- How are you calling / using the library? Command Palette
- What SQL language(s) does this apply to? Snowflake
- Which SQL Formatter version are you using? 1.6.0

**Comment by lulufredo** [comment]
Also wanted to say this is a fantastic extension—thank you so much for it!

**Comment by nene** [comment]
Hmm... I think what really should happen is that `QUALIFY` should be indented similarly to other clauses of the SELECT:

```sql
WHERE
    a.b IS NOT NULL
QUALIFY
    ROW_NUMBER() OVER ()...
```

**Comment by nene** [comment]
Looks like this issue has already been fixed almost a year ago in [12.0.4](https://github.com/sql-formatter-org/sql-formatter/releases/tag/v12.0.4) which should also be included to Prettier SQL VSCode plugin 1.6.0 that you claim to be using.

Anyway, that VSCode plugin is no more maintained (the maintainer has disappeared and I have no control over it). You should switch to [SQL Formatter VSCode](https://marketplace.visualstudio.com/items?itemName=ReneSaarsoo.sql-formatter-vsc) plugin.

**Comment by lulufredo** [comment]
Thanks for the note—I switched over to SQL Formatter VSC and it works!
